app/helpers/application_helper.rb

```diff
@@ -55,9 +55,12 @@ def hero_image_path(imgpath)
     end
   end
 
-  def fa_icon(icon_name, *additional_classes)
-    classes = ["fas", "fa-#{icon_name}"] + additional_classes
-    content_tag :i, "", class: classes.join(" ")
+  # FA supports several styles:
+  # fas = solid, fab = brand, far = regular, fal = light, fad = duotone
+  # https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use
+  def fa_icon(icon_name, *additional_classes, style: "fas")
+    classes = [style, "fa-#{icon_name}"] + additional_classes
+    tag.span("", class: classes)
   end
   alias_method :fas, :fa_icon
```

Comment:
Would it be better to replace this alias with

```ruby
  def fas(*args)
    fa_icon(*args, style: 'fas')
  end

  def fab(*args)
    fa_icon(*args, style: 'fab')
  end
```

Reply:
Makes sense. I think I'd go with `#fab_icon` though - `fas`/`fab` aren't too obvious.